pywebdriver/plugins/display_driver.py

```diff
@@ -68,7 +68,10 @@ def display_status_http():
         def get_status(self):
             try:
                 self.set_status('connected')
-                display_driver.push_task('send_text', [_(u'PyWebDriver'), _(u'PosBox Status')])
+                # When I use the POS, it regularly displays
+                # "PyWebDriver / PosBox Status" on the LCD !!!
+                # So I comment this line -- Alexis de Lattre
+                # display_driver.push_task('send_text', [_(u'PyWebDriver'), _(u'PosBox Status')])
```

Comment:
Thanks for this temporary path. :+1: 
could you just simplify, setting :   

    def get_status(self):
        self.set_status('connected') 

and merge my comment line 78 and your comment ?
(if you have a display regularly, it's because Odoo POS call get_status regularly...)